libraries/WiFi/examples/WiFiUDPClient/WiFiUDPClient.ino

```diff
@@ -0,0 +1,77 @@
+/*
+ *  This sketch sends random data over UDP on a ESP32 device
+ *
+ */
+#include <WiFi.h>
+#include <WiFiUdp.h>
+
+// WiFi network name and password:
+const char * networkName = "network_name";
+const char * networkPswd = "password";
+
+//IP address to send UDP data to:
+const char * udpAddress = "192.168.1.255";
+const int udpPort = 3333;
+
+//Are we currently connected?
+boolean connected = false;
+
+//The udp library class
+WiFiUDP udp;
+
+void setup(){
+  // Initilize hardware serial:
+  Serial.begin(115200);
+  
+  //Connect to the WiFi network
+  connectToWiFi(networkName, networkPswd);
+}
+
+void loop(){
+  //only send data when connected
+  if(connected){
+
+    //send one random byte
+    byte randomData = (byte) random(255);
+
+    //Send a packet
+    udp.beginPacket(udpAddress,udpPort);
+    udp.write(randomData);
```

Comment:
I changed this to:
```cpp
if(connected){
    //Send a packet
    udp.beginPacket(udpAddress,udpPort);
    udp.printf("seconds since boot: %u", millis()/1000);
    udp.endPacket();
}
```
and then used this simple python server to listen:
```python
import socket
import sys

try :
    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
    s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
except socket.error, msg :
    print 'Failed to create socket. Error Code : ' + str(msg[0]) + ' Message ' + msg[1]
    sys.exit()

try:
    s.bind(('', 3333))
except socket.error , msg:
    print 'Bind failed. Error: ' + str(msg[0]) + ': ' + msg[1]
    sys.exit()
     
print 'Server listening'

while 1:
    d = s.recvfrom(1024)
    data = d[0]
     
    if not data: 
        break
    
    print data.strip()
    
s.close()
```

You might want to add those to further enhance your example :)